### PR TITLE
Add VXU Parsing for CSVs

### PR DESCRIPTION
**Pull request by spencer-skylight**
Add VXU Parsing for CSVs

### CODE REVIEW

File: src/FunctionApps/python/tests/GenerateCSVs/test_vxu.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+from GenerateCSVs.vxu import vxu_to_csv
+import pytest
+
+
+@pytest.fixture()
+def bundle():
+    return json.load(open(pathlib.Path(__file__).parent / "assets" / "vxu.json"))
+
+
+def test_elr_to_csv(bundle):
```
Comment (graham-skylight) [inline src/FunctionApps/python/tests/GenerateCSVs/test_vxu.py:12]:
```suggestion
def test_vxu_to_csv(bundle):
```

File: src/FunctionApps/python/tests/GenerateCSVs/test_vxu.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    generated_rows = vxu_to_csv(bundle)
+    num_rows = 0
+    for row in generated_rows:
```
Comment (graham-skylight) [inline src/FunctionApps/python/tests/GenerateCSVs/test_vxu.py:15]: $0.02: I liked the thing you (I think?) did the other day where you converted into a set to do unordered list equality

**Review by graham-skylight — APPROVED** [review]
LGTM - just the test naming

File: src/FunctionApps/python/tests/GenerateCSVs/test_vxu.py (hunk 1 above)
Comment (spencer-skylight) [reply src/FunctionApps/python/tests/GenerateCSVs/test_vxu.py]: fixed

File: src/FunctionApps/python/tests/GenerateCSVs/test_vxu.py (hunk 2 above)
Comment (spencer-skylight) [reply src/FunctionApps/python/tests/GenerateCSVs/test_vxu.py]: I simplified by simply comparing to the full expected output.  I *think* list and set equality would both run in O(n), and the expected output ordering should be deterministic so I left it as an ordered list, but I might be missing something.

**Pull request merged**